API 34+ disallows creating or retrieving a PendingIntent with FLAG_MUTABLE

Hi,

I tried to clone this repo and start the example project but each times I got this error :
`MissingPluginException (MissingPluginException(No implementation found for method isSupported on channel uvccamera/native))`

I have put the 0.0.4 in version of the pubspec.yml :
```yaml
version: 0.0.4
```
Did I forgot to do something ?

This is my application logs :
```
Launching lib\main.dart on SM S911B in debug mode...
Warning: SDK processing. This version only understands SDK XML versions up to 3 but an SDK XML file of version 4 was encountered. This can happen if you use versions of Android Studio and the command-line tools that were released at different times.
√ Built build\app\outputs\flutter-apk\app-debug.apk
I/flutter_example(12113): Late-enabling -Xcheck:jni
I/flutter_example(12113): Using CollectorTypeCMC GC.
I/flutter (12113): [IMPORTANT:flutter/shell/platform/android/android_context_vk_impeller.cc(60)] Using the Impeller rendering backend (Vulkan).
W/flutter_example(12113): Accessing hidden method Landroid/view/accessibility/AccessibilityNodeInfo;->getSourceNodeId()J (unsupported,test-api, reflection, allowed)
W/flutter_example(12113): Accessing hidden method Landroid/view/accessibility/AccessibilityRecord;->getSourceNodeId()J (unsupported, reflection, allowed)
W/flutter_example(12113): Accessing hidden field Landroid/view/accessibility/AccessibilityNodeInfo;->mChildNodeIds:Landroid/util/LongArray; (unsupported, reflection, allowed)
W/flutter_example(12113): Accessing hidden method Landroid/util/LongArray;->get(I)J (unsupported, reflection, allowed)
I/flutter_example(12113): Compiler allocated 6943KB to compile void android.view.ViewRootImpl.performTraversals()
Connecting to VM Service at ws://127.0.0.1:61547/LCN0VTAwqJ8=/ws
Connected to the VM Service.
I/BLASTBufferQueue(12113): [SurfaceView[org.uvccamera.flutter_example/org.uvccamera.flutter_example.MainActivity]@0#1](f:0,a:0,s:0) onFrameAvailable the first frame is available
D/SurfaceView@823675(12113): 8533621 setAlpha: alpha=1.0
I/SurfaceView(12113): 8533621 Changes: creating=false format=false size=false visible=false alpha=true hint=false visible=false left=false top=false z=false attached=true lifecycleStrategy=false
I/SurfaceView@823675(12113): 8533621 Cur surface: Surface(name=null)/@0xf887227
I/SurfaceView@823675(12113): updateSurface: mVisible = true mSurface.isValid() = true
I/SurfaceView@823675(12113): updateSurface: mSurfaceCreated = true surfaceChanged = false visibleChanged = false
I/SurfaceView(12113): 8533621 surfaceRedrawNeeded
I/SurfaceView(12113): 8533621 finishedDrawing
V/SurfaceView@823675(12113): Layout: x=0 y=0 w=1080 h=2298, frame=Rect(0, 0 - 1080, 2298)
I/BLASTBufferQueue_Java(12113): update, w= 1080 h= 2340 mName = ViewRootImpl@4ee4c6b[MainActivity] mNativeObject= 0xb40000764adf7500 sc.mNativeObject= 0xb40000769316cd00 format= -3 caller= android.view.ViewRootImpl.updateBlastSurfaceIfNeeded:3074 android.view.ViewRootImpl.relayoutWindow:10224 android.view.ViewRootImpl.performTraversals:4167 android.view.ViewRootImpl.doTraversal:3345 android.view.ViewRootImpl$TraversalRunnable.run:11437 android.view.Choreographer$CallbackRecord.run:1690
I/ViewRootImpl@4ee4c6b[MainActivity](12113): Relayout returned: old=(0,0,1080,2340) new=(0,0,1080,2340) relayoutAsync=false req=(1080,2340)0 dur=8 res=0x401 s={true 0xb40000764ac9e800} ch=false seqId=0
I/ViewRootImpl@4ee4c6b[MainActivity](12113): updateBoundsLayer: t=android.view.SurfaceControl$Transaction@e0009c3 sc=Surface(name=Bounds for - org.uvccamera.flutter_example/org.uvccamera.flutter_example.MainActivity@0)/@0x8cfc240 frame=1
D/ViewRootImpl@4ee4c6b[MainActivity](12113): reportNextDraw android.view.ViewRootImpl.performTraversals:4781 android.view.ViewRootImpl.doTraversal:3345 android.view.ViewRootImpl$TraversalRunnable.run:11437 android.view.Choreographer$CallbackRecord.run:1690 android.view.Choreographer$CallbackRecord.run:1699
I/ViewRootImpl@4ee4c6b[MainActivity](12113): Setup new sync=wmsSync-ViewRootImpl@4ee4c6b[MainActivity]#2
I/ViewRootImpl@4ee4c6b[MainActivity](12113): Creating new active sync group ViewRootImpl@4ee4c6b[MainActivity]#3
I/SurfaceSyncGroup(12113): addLocalSync=ViewRootImpl@4ee4c6b[MainActivity]#3 to name=wmsSync-ViewRootImpl@4ee4c6b[MainActivity]#2, callers=android.window.SurfaceSyncGroup.add:431 android.window.SurfaceSyncGroup.add:392 android.window.SurfaceSyncGroup.add:340 android.view.ViewRootImpl.createSyncIfNeeded:4912 android.view.ViewRootImpl.performTraversals:4796 android.view.ViewRootImpl.doTraversal:3345
I/ViewRootImpl@4ee4c6b[MainActivity](12113): registerCallbacksForSync syncBuffer=false
D/SurfaceView(12113): 8533621 updateSurfacePosition RenderWorker, frameNr = 1, position = [0, 0, 1080, 2298] surfaceSize = 1080x2298
I/SurfaceView@823675(12113): uSP: rtp = Rect(0, 0 - 1080, 2298) rtsw = 1080 rtsh = 2298
I/SurfaceView@823675(12113): onSSPAndSRT: pl = 0 pt = 0 sx = 1.0 sy = 1.0
I/SurfaceView@823675(12113): aOrMT: ViewRootImpl@4ee4c6b[MainActivity] t = android.view.SurfaceControl$Transaction@86d42be fN = 1 android.view.SurfaceView.-$$Nest$mapplyOrMergeTransaction:0 android.view.SurfaceView$SurfaceViewPositionUpdateListener.positionChanged:1667 android.graphics.RenderNode$CompositePositionUpdateListener.positionChanged:369
I/ViewRootImpl@4ee4c6b[MainActivity](12113): mWNT: t=0xb40000764ac57800 mBlastBufferQueue=0xb40000764adf7500 fn= 1 mRenderHdrSdrRatio=1.0 caller= android.view.SurfaceView.applyOrMergeTransaction:1599 android.view.SurfaceView.-$$Nest$mapplyOrMergeTransaction:0 android.view.SurfaceView$SurfaceViewPositionUpdateListener.positionChanged:1667
I/ViewRootImpl@4ee4c6b[MainActivity](12113): Received frameDrawingCallback syncResult=0 frameNum=1.
I/ViewRootImpl@4ee4c6b[MainActivity](12113): mWNT: t=0xb40000764ac57f80 mBlastBufferQueue=0xb40000764adf7500 fn= 1 mRenderHdrSdrRatio=1.0 caller= android.view.ViewRootImpl$8.onFrameDraw:13946 android.view.ThreadedRenderer$1.onFrameDraw:792 <bottom of call stack>
I/ViewRootImpl@4ee4c6b[MainActivity](12113): Setting up sync and frameCommitCallback
I/BLASTBufferQueue(12113): [ViewRootImpl@4ee4c6b[MainActivity]#0](f:0,a:0,s:0) onFrameAvailable the first frame is available
I/ViewRootImpl@4ee4c6b[MainActivity](12113): Received frameCommittedCallback lastAttemptedDrawFrameNum=1 didProduceBuffer=true
I/SurfaceSyncGroup(12113): onTransactionReady mName=wmsSync-ViewRootImpl@4ee4c6b[MainActivity]#2 callback=187000185
D/OpenGLRenderer(12113): CFMS:: SetUp Pid : 12113    Tid : 12141
I/ViewRootImpl@4ee4c6b[MainActivity](12113): reportDrawFinished seqId=0
I/ViewRootImpl@4ee4c6b[MainActivity](12113): onDisplayChanged oldDisplayState=2 newDisplayState=2
I/ViewRootImpl@4ee4c6b[MainActivity](12113): handleWindowFocusChanged: 1 0 call from android.view.ViewRootImpl.-$$Nest$mhandleWindowFocusChanged:0
D/ViewRootImpl@4ee4c6b[MainActivity](12113): mThreadedRenderer.initializeIfNeeded()#2 mSurface={isValid=true 0xb40000764ac9e800}
D/InputMethodManagerUtils(12113): startInputInner - Id : 0
I/InputMethodManager(12113): startInputInner - IInputMethodManagerGlobalInvoker.startInputOrWindowGainedFocus
D/InputMethodManagerUtils(12113): startInputInner - Id : 0
I/InsetsSourceConsumer(12113): applyRequestedVisibilityToControl: visible=false, type=ime, host=org.uvccamera.flutter_example/org.uvccamera.flutter_example.MainActivity
I/ViewRootImpl@4ee4c6b[MainActivity](12113): onDisplayChanged oldDisplayState=2 newDisplayState=2
D/ProfileInstaller(12113): Installing profile for org.uvccamera.flutter_example
E/flutter (12113): [ERROR:flutter/runtime/dart_vm_initializer.cc(40)] Unhandled Exception: MissingPluginException(No implementation found for method isSupported on channel uvccamera/native)
E/flutter (12113): #0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:332:7)
E/flutter (12113): <asynchronous suspension>
E/flutter (12113): #1      UvcCameraPlatform.isSupported (package:uvccamera/src/uvccamera_platform.dart:31:20)
E/flutter (12113): <asynchronous suspension>
E/flutter (12113): #2      _UvcCameraDevicesScreenState.initState.<anonymous closure> (package:uvccamera_example/uvccamera_devices_screen.dart:24:34)
E/flutter (12113): <asynchronous suspension>
E/flutter (12113):

════════ Exception caught by services library ══════════════════════════════════
The following MissingPluginException was thrown while activating platform stream on channel uvccamera/device_events:
MissingPluginException(No implementation found for method listen on channel uvccamera/device_events)

When the exception was thrown, this was the stack:
#0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:332:7)
<asynchronous suspension>
#1      EventChannel.receiveBroadcastStream.<anonymous closure> (package:flutter/src/services/platform_channel.dart:676:9)
<asynchronous suspension>
════════════════════════════════════════════════════════════════════════════════
Application finished.

Exited (-1).
```

Flutter version : 3.27.1
Java SDK version : 17

Yes, like you said, I have the same issue with the CI apk.
After reading the logcat I found only this error :
```
12-23 14:12:37.578  1724  1724 D SurfaceFlinger:
12-23 14:12:37.585  4410  7285 I ViewRootImpl@61ebc72[LauncherActivity]: mWNT: t=0xb40000752a6be200 mBlastBufferQueue=0xb400007534b1f900 fn= 59 mRenderHdrSdrRatio=1.0 caller= com.android.systemui.shared.launcher.ViewRootImplCompat.mergeWithNextTransaction:4 com.honeyspace.transition.utils.SurfaceTransactionApplier.scheduleApply$lambda$1:26 com.honeyspace.transition.utils.SurfaceTransactionApplier.c:0
12-23 14:12:37.587  1724  1856 D RenderEngine: [SEC_GC_CMN_SF_EFFECTS] drawLayersInternal,1279, Rendering layer  at index 0
12-23 14:12:37.587  1724  1856 D RenderEngine: [SEC_GC_CMN_SF_EFFECTS] drawLayersInternal,1279, Rendering layer Wallpaper BBQ wrapper 5_system#77 at index 1
12-23 14:12:37.587  1724  1856 D RenderEngine: [SEC_GC_CMN_SF_EFFECTS] drawLayersInternal,1279, Rendering layer com.sec.android.app.launcher/com.sec.android.app.launcher.activities.LauncherActivity$_4410#101612 at index 2
12-23 14:12:37.593  4410  7284 I ViewRootImpl@61ebc72[LauncherActivity]: mWNT: t=0xb40000751d304180 mBlastBufferQueue=0xb400007534b1f900 fn= 60 mRenderHdrSdrRatio=1.0 caller= com.android.systemui.shared.launcher.ViewRootImplCompat.mergeWithNextTransaction:4 com.honeyspace.transition.utils.SurfaceTransactionApplier.scheduleApply$lambda$1:26 com.honeyspace.transition.utils.SurfaceTransactionApplier.c:0
12-23 14:12:37.600 25825 25825 V d2.k    : onAttachedToEngine
12-23 14:12:37.602  4410  7285 I ViewRootImpl@61ebc72[LauncherActivity]: mWNT: t=0xb40000751d304300 mBlastBufferQueue=0xb400007534b1f900 fn= 61 mRenderHdrSdrRatio=1.0 caller= com.android.systemui.shared.launcher.ViewRootImplCompat.mergeWithNextTransaction:4 com.honeyspace.transition.utils.SurfaceTransactionApplier.scheduleApply$lambda$1:26 com.honeyspace.transition.utils.SurfaceTransactionApplier.c:0
12-23 14:12:37.604 25825 25825 D CompatibilityChangeReporter: Compat change id reported: 160794467; UID 11002; state: ENABLED
12-23 14:12:37.604 25825 25825 D CompatibilityChangeReporter: Compat change id reported: 236704164; UID 11002; state: ENABLED
12-23 14:12:37.604  2538  6862 D CompatibilityChangeReporter: Compat change id reported: 236704164; UID 11002; state: ENABLED
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant: Error registering plugin uvccamera, org.uvccamera.flutter.UvcCameraPlugin
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant: java.lang.IllegalArgumentException: org.uvccamera.flutter_example: Targeting U+ (version 34 and above) disallows creating or retrieving a PendingIntent with FLAG_MUTABLE, an implicit Intent within and without FLAG_NO_CREATE and FLAG_ALLOW_UNSAFE_IMPLICIT_INTENT for security reasons. To retrieve an already existing PendingIntent, use FLAG_NO_CREATE, however, to create a new PendingIntent with an implicit Intent use FLAG_IMMUTABLE.
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.os.Parcel.createExceptionOrNull(Parcel.java:3073)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.os.Parcel.createException(Parcel.java:3053)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.os.Parcel.readException(Parcel.java:3036)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.os.Parcel.readException(Parcel.java:2978)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.app.IActivityManager$Stub$Proxy.getIntentSenderWithFeature(IActivityManager.java:7055)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.app.PendingIntent.getBroadcastAsUser(PendingIntent.java:752)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.app.PendingIntent.getBroadcast(PendingIntent.java:735)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at d2.j.<init>(SourceFile:88)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at d2.k.d(SourceFile:49)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at s0.e.a(SourceFile:84)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at io.flutter.plugins.GeneratedPluginRegistrant.registerWith(SourceFile:44)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at java.lang.reflect.Method.invoke(Native Method)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at h1.f.V(SourceFile:22)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at org.uvccamera.flutter_example.MainActivity.F(SourceFile:8)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at org.uvccamera.flutter_example.MainActivity.J(SourceFile:369)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at org.uvccamera.flutter_example.MainActivity.onCreate(SourceFile:1)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.app.Activity.performCreate(Activity.java:9001)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.app.Activity.performCreate(Activity.java:8970)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1456)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4154)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4330)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2693)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.os.Handler.dispatchMessage(Handler.java:106)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.os.Looper.loopOnce(Looper.java:230)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.os.Looper.loop(Looper.java:319)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.app.ActivityThread.main(ActivityThread.java:9063)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at java.lang.reflect.Method.invoke(Native Method)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:588)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1103)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant: Caused by: android.os.RemoteException: Remote stack trace:
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at com.android.server.am.ActivityManagerService.getIntentSenderWithFeatureAsApp(ActivityManagerService.java:6512)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at com.android.server.am.ActivityManagerService.getIntentSenderWithFeature(ActivityManagerService.java:6455)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.app.IActivityManager$Stub.onTransact$getIntentSenderWithFeature$(IActivityManager.java:11997)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at android.app.IActivityManager$Stub.onTransact(IActivityManager.java:3422)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:     at com.android.server.am.ActivityManagerService.onTransact(ActivityManagerService.java:3202)
12-23 14:12:37.605 25825 25825 E GeneratedPluginRegistrant:
12-23 14:12:37.606 25825 25825 D SurfaceView@86189ac: 140609964 setAlpha: alpha=0.0
12-23 14:12:37.606 25825 25825 D SurfaceView@86189ac: 140609964 updateSurface: has no frame
12-23 14:12:37.610  4410  7284 I ViewRootImpl@61ebc72[LauncherActivity]: mWNT: t=0xb40000751d304480 mBlastBufferQueue=0xb400007534b1f900 fn= 62 mRenderHdrSdrRatio=1.0 caller= com.android.systemui.shared.launcher.ViewRootImplCompat.mergeWithNextTransaction:4 com.honeyspace.transition.utils.SurfaceTransactionApplier.scheduleApply$lambda$1:26 com.honeyspace.transition.utils.SurfaceTransactionApplier.c:0
12-23 14:12:37.612 25825 25825 I DecorView: setWindowBackground: isPopOver=false color=ff000000 d=android.graphics.drawable.ColorDrawable@fa4ccae
12-23 14:12:37.615 25825 25825 D CompatibilityChangeReporter: Compat change id reported: 237531167; UID 11002; state: DISABLED
12-23 14:12:37.616 25825 25851 D NativeCustomFrequencyManager: [NativeCFMS] BpCustomFrequencyManager::BpCustomFrequencyManager()
12-23 14:12:37.618  4410  7285 I ViewRootImpl@61ebc72[LauncherActivity]: mWNT: t=0xb40000751d304600 mBlastBufferQueue=0xb400007534b1f900 fn= 63 mRenderHdrSdrRatio=1.0 caller= com.android.systemui.shared.launcher.ViewRootImplCompat.mergeWithNextTransaction:4 com.honeyspace.transition.utils.SurfaceTransactionApplier.scheduleApply$lambda$1:26 com.honeyspace.transition.utils.SurfaceTransactionApplier.c:0
12-23 14:12:37.618  2538  6862 D InputTransport: Input channel constructed: 'c0dea24', fd=966
12-23 14:12:37.618  2538  6862 D InputTransport: Input channel constructed: 'c0dea24', fd=1016
12-23 14:12:37.618  2538  6862 D InputTransport: Input channel constructed: 'c0dea24', fd=1019
12-23 14:12:37.619  1724  1836 I SurfaceFlinger: id=101629 createSurf, flag=84004, c0dea24 org.uvccamera.flutter_example/org.uvccamera.flutter_example.MainActivity#101629
12-23 14:12:37.619  2538  6862 D RestrictionPolicy: isScreenCaptureEnabled : ret=true userId=0
12-23 14:12:37.619  2538  6862 D InputTransport: Input channel destroyed: 'c0dea24', fd=1019
12-23 14:12:37.620 25825 25825 I InsetsController: onStateChanged: host=org.uvccamera.flutter_example/org.uvccamera.flutter_example.MainActivity, from=android.view.ViewRootImpl.setView:1797, state=InsetsState: {mDisplayFrame=Rect(0, 0 - 1080, 2340), mDisplayCutout=DisplayCutout{insets=Rect(0, 81 - 0, 0) waterfall=Insets{left=0, top=0, right=0, bottom=0} boundingRect={Bounds=[Rect(0, 0 - 0, 0), Rect(512, 0 - 568, 81), Rect(0, 0 - 0, 0), Rect(0, 0 - 0, 0)]} cutoutPathParserInfo={CutoutPathParserInfo{displayWidth=1080 displayHeight=2340 physicalDisplayWidth=1080 physicalDisplayHeight=2340 density={3.0} cutoutSpec={M 0,0 H -9.333333333333333‬ V 27 H 9.333333333333333‬ V 0 H 0 Z @dp} rotation={0} scale={1.0} physicalPixelDisplaySizeRatio={1.0}}}}, mRoundedCor rners=RoundedCorners{[RoundedCorner{position=TopLeft, radius=96, center=Point(96, 96)}, RoundedCorner{position=TopRight, radius=96, center=Point(984, 96)}, RoundedCorner{position=BottomRight, radius=96, center=Point(984, 2244)}, RoundedCorner{position=BottomLeft, radius=96, center=Point(96, 2244)}]}  mRoundedCornerFrame=Rect(0, 0 - 1080, 2340), mPrivacyIndicatorBounds=PrivacyIndicatorBounds {static bounds=Rect(956, 0 - 1080, 81) rotation=0}, mDisplayShape=DisplayShape{ spec=-311912193 displayWidth=1080 displayHeight=2340 physicalPixelDisplaySizeRatio=1.0 rotation=0 offsetX=0 offsetY=0 scale=1.0}, mSources= { InsetsSource: {3 mType=ime mFrame=[0,0][0,0] mVisible=false mFlags=[]}, InsetsSource: {27 mType=displayCutout mFrame=[0,0][1080,81] mVisible=true mFlags=[]}, InsetsSource: {61d50000 mType=statusBars mFrame=[0,0][1080,81] mVisible=true mFlags=[]}, InsetsSource: {61d50005 mType=mandatorySystemGestures mFrame=[0,0][1080,115] mVisible=true mFlags=[]}, InsetsSource: {61d50006 mType=tappableElement mFrame=[0,0][1080,81] mVisible=true mFlags=[]}, InsetsSource: {71cb0001 mType=navigationBars mFrame=[0,2298][1080,2340] mVisible=true mFlags=[SUPPRESS_SCRIM]}, InsetsSource: {71cb0004 mType=systemGestures mFrame=[0,0][84,2340] mVisible=true mFlags=[]}, InsetsSource: {71cb0005 mType=mandatorySystemGestures mFrame=[0,2250][1080,2340] mVisible=true mFlags=[]}, InsetsSource: {71cb0006 mType=tappableElement mFrame=[0,0][0,0] mVisible=true mFlags=[]}, InsetsSource: {71cb0024 mType=systemGestures mFrame=[996,0][1080,2340] mVisible=true mFlags=[]} }
12-23 14:12:37.620 25825 25825 I ViewRootImpl@1f505ba[MainActivity]: synced displayState. AttachInfo displayState=2
12-23 14:12:37.620 25825 25825 I ViewRootImpl@1f505ba[MainActivity]: setView = com.android.internal.policy.DecorView@3ddc386 TM=true
12-23 14:12:37.620 25825 25825 I IDS_TAG : Starting IDS observe window
12-23 14:12:37.620 25825 25825 I IDS_TAG : Getting Shared Preference for android.app.Application@6a34447 uid = 11002
12-23 14:12:37.622 25825 25825 I IDS_TAG : App android.app.Application@6a34447 has not finished training
12-23 14:12:37.622 25825 25851 D OpenGLRenderer: HWUI - treat SMPTE_170M as sRGB
12-23 14:12:37.622 25825 25825 I IDS_TAG : Closing IDS observe window
12-23 14:12:37.622 25825 25825 I IDS_TAG : Getting Shared Preference for android.app.Application@6a34447 uid = 11002
12-23 14:12:37.623 25825 25825 I IDS_TAG : IDS count updated to 3 for android.app.Application@6a34447
12-23 14:12:37.624 25825 25825 I SurfaceView@86189ac: onWindowVisibilityChanged(0) true r0.l{86189ac V.E...... ......I. 0,0-0,0} of ViewRootImpl@1f505ba[MainActivity]
12-23 14:12:37.624 25825 25825 D SurfaceView@86189ac: 140609964 updateSurface: has no frame
12-23 14:12:37.625  2538  6833 V WindowManager: Relayout Window{c0dea24 u0 org.uvccamera.flutter_example/org.uvccamera.flutter_example.MainActivity}: viewVisibility=0 req=1080x2340 ty=1 d0   
12-23 14:12:37.625  2538  6833 D RestrictionPolicy: isScreenCaptureEnabled : ret=true userId=0
12-23 14:12:37.625  1724  1837 I SurfaceFlinger: id=101630 createSurf, flag=44004, org.uvccamera.flutter_example/org.uvccamera.flutter_example.MainActivity$_25825#101630
12-23 14:12:37.626  2538  6833 D WindowManager: makeSurface duration=0 name=org.uvccamera.flutter_example/org.uvccamera.flutter_example.MainActivity$_25825
12-23 14:12:37.626  2538  6833 I WindowManager: Cancelling animation restarting=true, leash=Surface(name=Surface(name=eeaf924 NavigationBar0)/@0xc5a67fa - animation-leash of insets_animation)/@0xf2d295f
12-23 14:12:37.626  2538  6833 I WindowManager: Reparenting to original parent: Surface(name=WindowToken{ec6a48c type=2019 android.os.BinderProxy@a457663})/@0x388a1ab, destroy=false, surface=Surface(name=eeaf924 NavigationBar0)/@0xc5a67fa
12-23 14:12:37.626  2538  6833 I WindowManager: Reparenting to leash, surface=Surface(name=eeaf924 NavigationBar0)/@0xc5a67fa, leashParent=Surface(name=WindowToken{ec6a48c type=2019 android.os.BinderProxy@a457663})/@0x388a1ab
12-23 14:12:37.626  4410  7284 I ViewRootImpl@61ebc72[LauncherActivity]: mWNT: t=0xb40000751d304780 mBlastBufferQueue=0xb400007534b1f900 fn= 64 mRenderHdrSdrRatio=1.0 caller= com.android.systemui.shared.launcher.ViewRootImplCompat.mergeWithNextTransaction:4 com.honeyspace.transition.utils.SurfaceTransactionApplier.scheduleApply$lambda$1:26 com.honeyspace.transition.utils.SurfaceTransactionApplier.c:0
12-23 14:12:37.626  1724  1837 I SurfaceFlinger: id=101631 createSurf, flag=24000, Surface(name=eeaf924 NavigationBar0)/@0xc5a67fa - animation-leash of insets_animation#101631
12-23 14:12:37.627  2538  6833 D WindowManager: makeSurface duration=0 leash=Surface(name=Surface(name=eeaf924 NavigationBar0)/@0xc5a67fa - animation-leash of insets_animation)/@0x8612353    
12-23 14:12:37.627  2538  6833 D InsetsSourceProvider: updateControlForTarget: control=InsetsSourceControl: {71cb0001 mType=navigationBars initiallyVisible mSurfacePosition=Point(0, 2205) mInsetsHint=Insets{left=0, top=0, right=0, bottom=42}}, target=Window{f2317cd u0 Splash Screen org.uvccamera.flutter_example}, from=com.android.server.wm.InsetsStateController.onControlTargetChanged:363 com.android.server.wm.InsetsStateController.onBarControlTargetChanged:332 com.android.server.wm.InsetsPolicy.updateBarControlTarget:173 com.android.server.wm.InsetsPolicy.updateSystemBars:843 com.android.server.wm.DisplayPolicy.updateSystemBarsLw:3484
12-23 14:12:37.627  2538  6833 I WindowManager: Cancelling animation restarting=true, leash=Surface(name=Surface(name=fb0d745 StatusBar)/@0xdd709c6 - animation-leash of insets_animation)/@0x4633475
12-23 14:12:37.627  2538  6833 I WindowManager: Reparenting to original parent: Surface(name=WindowToken{cead78e type=2000 android.os.BinderProxy@69ff690})/@0xeadad87, destroy=false, surface=Surface(name=fb0d745 StatusBar)/@0xdd709c6
12-23 14:12:37.627  2538  6833 I WindowManager: Reparenting to leash, surface=Surface(name=fb0d745 StatusBar)/@0xdd709c6, leashParent=Surface(name=WindowToken{cead78e type=2000 android.os.BinderProxy@69ff690})/@0xeadad87
12-23 14:12:37.627  1724  1837 I SurfaceFlinger: id=101632 createSurf, flag=24000, Surface(name=fb0d745 StatusBar)/@0xdd709c6 - animation-leash of insets_animation#101632
12-23 14:12:37.627  2538  6833 D WindowManager: makeSurface duration=1 leash=Surface(name=Surface(name=fb0d745 StatusBar)/@0xdd709c6 - animation-leash of insets_animation)/@0x3798189
12-23 14:12:37.627  2538  6833 D InsetsSourceProvider: updateControlForTarget: control=InsetsSourceControl: {61d50000 mType=statusBars initiallyVisible mSurfacePosition=Point(0, 0) mInsetsHint=Insets{left=0, top=81, right=0, bottom=0}}, target=Window{f2317cd u0 Splash Screen org.uvccamera.flutter_example}, from=com.android.server.wm.InsetsStateController.onControlTargetChanged:363 com.android.server.wm.InsetsStateController.onBarControlTargetChanged:329 com.android.server.wm.InsetsPolicy.updateBarControlTarget:173 com.android.server.wm.InsetsPolicy.updateSystemBars:843 com.android.server.wm.DisplayPolicy.updateSystemBarsLw:3484
12-23 14:12:37.627 29433 29433 I cixt    : onPause
12-23 14:12:37.627  2538  6833 D WindowManager: updateSystemBarAttributes: displayId=0, win=Window{f2317cd u0 Splash Screen org.uvccamera.flutter_example}, navColorWin=Window{f2317cd u0 Splash Screen org.uvccamera.flutter_example}, focusedCanBeNavColorWin=false, behavior=1, appearance=0, statusBarAppearanceRegions=[AppearanceRegion{ bounds=[0,0][1080,2340]}], requestedVisibilities=-9, from=com.android.server.wm.DisplayPolicy.finishPostLayoutPolicyLw:2339 com.android.server.wm.DisplayContent.applySurfaceChangesTransaction:6181 com.android.server.wm.RootWindowContainer.applySurfaceChangesTransaction:1183
12-23 14:12:37.629  3940  3940 D SysUiState: SysUiState changed: old=0x10000000 new=0x10020000
```

--- a/lib/src/main/java/com/serenegiant/usb/USBMonitor.java
+++ b/lib/src/main/java/com/serenegiant/usb/USBMonitor.java
@@ -168,7 +168,13 @@ public final class USBMonitor {
 			if (DEBUG) Log.i(TAG, "register:");
 			final Context context = mWeakContext.get();
 			if (context != null) {
-				mPermissionIntent = PendingIntent.getBroadcast(context, 0, new Intent(ACTION_USB_PERMISSION), PendingIntent.FLAG_MUTABLE);
+				final var permissionIntent = new Intent(ACTION_USB_PERMISSION);
+				permissionIntent.setPackage(context.getPackageName());
+				if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+					mPermissionIntent = PendingIntent.getBroadcast(context, 0, permissionIntent, PendingIntent.FLAG_MUTABLE);
+				} else {
+					mPermissionIntent = PendingIntent.getBroadcast(context, 0, permissionIntent, 0);
+				}
 				final IntentFilter filter = new IntentFilter(ACTION_USB_PERMISSION);
 				// ACTION_USB_DEVICE_ATTACHED never comes on some devices so it should not be added here
 				filter.addAction(UsbManager.ACTION_USB_DEVICE_DETACHED);

